commons/active/banking/banking_attempt.avsc

```diff
@@ -0,0 +1,30 @@
+{
+  "namespace": "org.radarcns.active.banking",
+  "type": "record",
+  "name": "BankingAttempt",
+  "doc": "A Banking App Attempt to complete a transaction, which can contain a PIN, an Amount and a Confirm/Cancel Transaction event. It can also time out or end.",
+  "fields": [
+    { "name": "uid", "type": "string", "doc": "Unique ID for an Attempt."},
+    { "name": "time", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "timeCompleted", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "duration", "type": "double", "doc": "Duration of the activity (seconds)."},
+    { "name": "isComplete", "type":"boolean", "doc": "Describes if the Attempt was completed or not."},
```

Comment:
is this simply a proxy for `isComplete = result ~= *_CONFIRMED || result ~= *_CANCELLED`? Perhaps this variable be removed then.

Reply:
isComplete refers to if the attempt was finished succesfully or if it was Exited, Timed_out. So, we included it as a more high level variable.
So, for example, if the result is *_cancelled, it is still completed in that way.

Reply:
I'm not a fan of this duplication, but it's OK. In the doc, can you please describe explicitly that this is how the variable is computed (e.g. `True if the result is CORRECT_CONFIRMED, WRONG_CONFIRMED, CORRECT_CANCELLED or WRONG_CANCELLED. False otherwise.`)?

Reply:
OK thank you.